docs/6-Efficient-Bundling.stories.mdx

```diff
@@ -0,0 +1,122 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Efficient Bundling" />
+
+# Efficient Bundling
+
+This section describes some options on how to optimize your bundle size.
+
+**Following these steps require you to modify the default `create-react-app` setup. This might result in broken applications!.**<br />
```

Comment:
```suggestion
**Following these steps require you to modify the default `create-react-app` setup. This might result in broken applications!**<br />
```